dojo/tools/sarif/parser.py

```diff
@@ -185,18 +196,19 @@ def get_item(result, rules, artifacts, run_date):
     # we add a special 'None' case if there is no CWE
     cwes = [0]
     if rule is not None:
-        cwes_extracted = get_rule_cwes(result)
+        cwes_extracted = get_rule_cwes(rule)
+        cwes_properties_extracted = get_result_cwes_properties(result)
```

Comment:
`get_result_cwes_properties` is now only called when there are rules. Wouldn't it make sense to get CWEs from the results if there are no rules as well?

Reply:
Modified following your comment